openbb_platform/core/openbb_core/app/static/utils/decorators.py

```diff
@@ -44,14 +44,36 @@ def exception_handler(func: Callable[P, R]) -> Callable[P, R]:
     """Handle exceptions, attempting to focus on the last call from the traceback."""
 
     @wraps(func)
-    def wrapper(*args, **kwargs):
+    def wrapper(*f_args, **f_kwargs):
         try:
-            return func(*args, **kwargs)
-        except Exception as e:
+            return func(*f_args, **f_kwargs)
+        except (ValidationError, Exception) as e:
             if Env().DEBUG_MODE:
                 raise
+            if isinstance(e, ValidationError):
+                error_list = []
+
+                validation_error = f"{e.error_count()} validations errors in {e.title}"
+                for error in e.errors():
+                    arg_error = f"Arg {error['loc'][0]} ->\n"
+                    error_details = (
+                        f"  {error['msg']} "
+                        f"[validation_error_type={error['type']}, "
+                        f"input_type={type(error['input']).__name__}, "
+                        f"input_value={error['input']}]\n"
+                    )
+                    error_info = f"    For further information visit {error['url']}\n"
+                    error_list.append(arg_error + error_details + error_info)
+
+                error_list.insert(0, validation_error)
+                error_str = "\n".join(error_list)
+                raise OpenBBError(
+                    f"\nType -> ValidationError \n\nDetails -> {error_str}"
+                ) from None
```

Comment:
imo, this should be `from e`, otherwise it will point to the decorator instead of the actual code that's breaking

Reply:
If we do `from e` then we are adding the context of the original exception which is wrapped by `OpenBBError`. To reduce the traceback call we use `from None` so that the context of the original exception is not used and so that is suppressed.

Reply:
let's keep it that way until we find a good solution for the traceback suppression, then!